pilot.py

```diff
@@ -65,6 +66,18 @@ def main():
                             required=True,
                             help='MANDATORY: queue name (e.g., AGLT2_TEST-condor')
 
+    # SSL certificates
+    arg_parser.add_argument('--cacert',
+                            dest='cacert',
+                            default=None,
```

Comment:
I was doing at-the-init defaults-obtaining thing, but it only works in 2.7 and higher, with use of some lambdas, see pilot-2.0 code.